src/game/bullet-collision.ts

```diff
@@ -39,11 +40,7 @@ export function registerBulletCollisionSystem(em: EntityManager) {
           );
           for (let otherBullet of otherBullets) {
             if (otherBullet) {
-              resources.detectedEvents.raise({
-                type: "bullet-bullet",
-                entities: [o.id, otherBullet.id],
-                extra: null,
-              });
+              raiseBulletBullet([o, otherBullet]);
```

Comment:
notice how much nicer it is to raise events

Reply:
yeah this is sick